scss/mixins/_soe_breakpoints.scss

```diff
@@ -3,7 +3,9 @@
 // MAX-WIDTH BREAKPOINTS
 
 @mixin breakpoint-max($point) {
-  @if $point == large {
+  @if $point == x-large {
+    @media (max-width: 1250px) { @content; }
+  }@else if $point == large {
```

Comment:
Not sure why the linter missed this but there should be a space after the curly brace and the else statement.

Reply:
@sherakama fixed